### ISSUE TITLE
Can not add nutris

### ISSUE THREAD

**Issue by poltrian** [nextcloud/cookbook#723]
**Description**

Create a recipe and select the nutritional values. Nothing happens

**Browser**
Firefox

**Versions**
Nextcloud server version: 21
Cookbook version: lastes
Database system: MariaDB/

**Comment by christianlupus** [comment]
Unfortunately, your issue is not complete. Some major parts are missing.

What steps are you performing exactly? What do you expect to happen? What is happening instead?

Adding some screenshots might be very helpful to understand the issue.

**Comment by poltrian** [comment]
hi
i try to edit 
I mark an option but it is not accepted

I'm trying to edit a recipe or create a new one by hand. Then I select the nutritional values, mark them and nothing happens. I reload the recipe (without saving it) it works

![Bildschirmfoto vom 2021-05-27 08-26-27](https://user-images.githubusercontent.com/50879520/119777111-0dd81880-bec6-11eb-88f2-e1ce770c81a9.png)

**Comment by seyfeb** [comment]
Can you open the developer tools in the browser, check in the Console tab for any error messages, and share them here? Which browser(s) have you tried?

**Comment by poltrian** [comment]
`Structured Ingestion ping failure with error: error 3 PingCentre.jsm:202
    sendStructuredIngestionPing resource:///modules/PingCentre.jsm:202
[Exception... "Favicon at "https://www.schwarzgrueneszebra.de/favicon.ico" failed to load: Not Found."  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource:///modules/FaviconLoader.jsm :: onStopRequest :: line 253"  data: no] FaviconLoader.jsm:253:22`

Lastest Firefox & Chrome

**Comment by poltrian** [comment]
https://user-images.githubusercontent.com/50879520/119782094-0f0c4400-becc-11eb-92b8-79efe5dcdb12.mp4

**Comment by GambaJo** [comment]
I have the same problem. But only with old (weeks or moths) recipes. When I create a new one, I can add nutris.

**Comment by christianlupus** [comment]
@poltrian can you confirm the behavior of @GambaJo? I think this is a different issue as it only affects old recipes.

Apart from that, is there anything logged in the NC admin logs?
If this does not get to the point, you can look in the development tools for the save command and see if 

> I mark an option but it is not accepted
> 
> I'm trying to edit a recipe or create a new one by hand. Then I select the nutritional values, mark them and nothing happens. I reload the recipe (without saving it) it works

To give you a hint of what would be useful, I am guessing a bit. So am I getting you right?
1. Create a new recipe
2. Scrolling to the nutritions
3. Click on the left field to select the type of information to be saved
4. Click on one entry (e.g. fat)
5. Enter some value into the right field
6. Save the recipe
7. Reopen the edit view and scroll to nutritions

After step 4 no entry is saved in the left-sided input?
Honestly, I am very unsure if I understand the wrong behavior. As this is absolutely required in order to tackle this problem, it might be a good idea to add a series of screenshots (with comments) or even a short video that shows the problem.

@GambaJo do you have some recipes that are causing trouble? If yes, could you share the corresponding JSON with us?
It might be a good idea to open a new issue to separate the issues correctly.

**Comment by GambaJo** [comment]
[Banh Mi.zip](https://github.com/nextcloud/cookbook/files/6872337/Banh.Mi.zip)
[Gänsebrust.zip](https://github.com/nextcloud/cookbook/files/6872338/Gansebrust.zip)
If you need more, tell me please.

**Comment by christianlupus** [comment]
uups, I forgot to remove the stale bot.

**Comment by GambaJo** [comment]
> 
> 
> @poltrian @GambaJo I just tried to reproduce the problem and I failed to do so. I could modify the nutrition information of the provided example recipes and create new recipes. Can you please check, if this is still valid with the upcoming 0.9.6 version (I will publish a release in a few minutes)?

I tried with 0.9.6 and NC 22.2.0 and it works now. Thx!
